PWG/EMCAL/EMCALtasks/AliEmcalCorrectionCellTimeCalib.cxx

```diff
@@ -163,16 +164,16 @@ Int_t AliEmcalCorrectionCellTimeCalib::InitTimeCalibration()
   { // Else choose the one in the $ALICE_PHYSICS directory
     AliInfo("Loading time calibration OADB from $ALICE_PHYSICS/OADB/EMCAL");
     
-    TFile *fbad=new TFile("$ALICE_PHYSICS/OADB/EMCAL/EMCALTimeCalib.root","read");
+    TFile *fbad=new TFile(AliDataFile::GetFileNameOADB("EMCAL/EMCALTimeCalib.root").data(),"read");
```

Comment:
std::unique_ptr<TFile> ?

Reply:
I am not sure what you mean with your comment. If something is wrong then let me know.

Reply:
I mean

```
std::unique_ptr<TFile> fBad(new TFile(...));
```

This will close and delete the file at tha end of the function automatically.

Reply:
Feel free to implement this in a separate PR. I will not make these changes for this PR as I only intended to change the path to the OADB.

Reply:
@mfasDa , these were inherited from the old correction framework. We'd be happy to get a PR here, but otherwise this isn't a huge priority for us